stl/inc/chrono

```diff
@@ -3389,12 +3389,15 @@ namespace chrono {
                     if (_Subsecond) {
                         using _CastedType = duration<int64_t, typename _DurationType::period>;
                         const _SubsecondType _Sub{*_Subsecond};
-                        if (treat_as_floating_point_v<_DurationType> //
+#pragma warning(push)
+#pragma warning(disable : 4127) // conditional expression is constant
+                        if (treat_as_floating_point_v<typename _DurationType::rep> //

```

Comment:
While this is correct, we could avoid the need for push-disable-pop by refactoring this with `if constexpr` and De Morgan's law.